sw/device/lib/drivers/dma/dma.c

```diff
@@ -697,11 +706,14 @@ dma_config_flags_t dma_load_transaction( dma_trans_t *p_trans )
                     DMA_PTR_INC_SRC_PTR_INC_OFFSET );
    
    
-    //printf("dst inc - ");
-    write_register(  get_increment_b( dma_cb.trans->dst ), 
-                    DMA_PTR_INC_REG_OFFSET, 
-                    DMA_PTR_INC_DST_PTR_INC_MASK,
-                    DMA_PTR_INC_DST_PTR_INC_OFFSET );
+
+    if(dma_cb.trans->mode != DMA_TRANS_MODE_ADDRESS) {
+        //printf("dst inc - ");
+        write_register(  get_increment_b( dma_cb.trans->dst ),
+                        DMA_PTR_INC_REG_OFFSET,
+                        DMA_PTR_INC_DST_PTR_INC_MASK,
+                        DMA_PTR_INC_DST_PTR_INC_OFFSET );
+    }
```

Comment:
I understand that in _ADDRESS MODE_ the destination increment is ignored right? 
I will add this to the documentation, but can we add a comment here to make it explicit?

Reply:
correct